ebos/equil/initstateequil.hh

```diff
@@ -166,14 +169,14 @@ public:
 
 private:
     const double temp_;
+    const double saltConcentration_;
     const int pvtRegionIdx_;
     const double g_;
 
     double
     density(const double press) const
     {
-        double saltConcentration = 0.0; // TODO allow for non-zero initial salt concentration
-        double rho = FluidSystem::waterPvt().inverseFormationVolumeFactor(pvtRegionIdx_, temp_, press, saltConcentration);
+        double rho = FluidSystem::waterPvt().inverseFormationVolumeFactor(pvtRegionIdx_, temp_, press, saltConcentration_);
```

Comment:
This will use a constant salt concentration for all cells in an equilibration region.  Is that intended?  If not, do you want to fix it now or at a later time?

Reply:
The salt concentration should be pr. cell. I miss read the code. Thanks for noticing. I will update the code accordingly.